script.js

```diff
@@ -0,0 +1,83 @@
+const form = document.querySelector('.form');
+class Book {
```

Comment:
- [ ]  it would be better if you the `luxon` library for display the date time in order to match the requirements 
<img width="1040" alt="Screen Shot 2022-09-26 at 18 50 20" src="https://user-images.githubusercontent.com/61790587/192346233-41e32e80-bec0-4cc5-b7a3-8a8128e8f3a9.png">